po/es.po

```diff
@@ -0,0 +1,747 @@
+# Spanish translations for connman-gtk package.
+# Copyright (C) 2015 THE connman-gtk'S COPYRIGHT HOLDER
```

Comment:
I think the year should be '2018' here since this file is created on this year.

Reply:
The suggested corrections are already ready, in 'fsid' it was an error when transcribing.